tests/console/check_package_version.pm

```diff
@@ -73,7 +75,11 @@ sub run {
     }
 
     assert_script_run('python3 --version', 30);
-    die "there are failed packages" if @failed_pcks;
+    record_soft_failure "poo#54257, jsc#7146 some failed packages" if @failed_pcks;
```

Comment:
Maybe we can store the jira number into the package struct, then we can softfail print out which Jira failed.